src/payment/strategies/amazon-pay-v2/amazon-pay-v2-payment-strategy.ts

```diff
@@ -148,13 +144,15 @@ export default class AmazonPayV2PaymentStrategy implements PaymentStrategy {
         }), { queueId: 'widgetInteraction' });
     }
 
-    private _createSignInButton(containerId: string, paymentMethod: PaymentMethod): HTMLElement {
-        const container = document.getElementById(containerId);
+    private _createContainer(): HTMLDivElement {
+        const container = document.createElement('div');
+        container.id = 'AmazonPayButton';
```

Comment:
You should check to make sure the container is not created already. Otherwise you'll end up with multiple containers with the same ID. That's not good because Amazon SDK won't be able the select the right one.